x-pack/plugins/file_upload/server/utils/runtime_field_utils.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { estypes } from '@elastic/elasticsearch';
+import { RUNTIME_FIELD_TYPES } from '../../../../../src/plugins/data/common/index_patterns';
+
+type RuntimeType = typeof RUNTIME_FIELD_TYPES[number];
+
+export const isPopulatedObject = <U extends string = string>(
```

Comment:
this has already been added to common
`x-pack/plugins/data_visualizer/common/utils/object_utils.ts `

There would be no harm in moving the two functions below to common too.

Reply:
ah ok, sorry i missed the file path and didn't realise this was in a different plugin.